analysis/dataset_definition.py

```diff
@@ -19,15 +19,15 @@
 # Inclusion/exclusion criteria:
 
 ## Include people alive
```

Comment:
If you use the approach I suggested on Slack (only extract data for people who died), here you would identify people whose TPP or ONS date of death is NOT null